bokeh/io.py

```diff
@@ -303,7 +303,7 @@ def _show_notebook_app_with_state(app, state, app_path, notebook_url):
     loop = IOLoop.current()
     server = Server({app_path: app}, io_loop=loop, port=0,  allow_websocket_origin=[notebook_url])
     server.start()
-    script = autoload_server(url='http://127.0.0.1:%d%s' % (server.port, app_path))
+    script = server_document(url='http://%s:%d%s' % (notebook_url.split(':')[0], server.port, app_path))
```

Comment:
very minor thing but one of the reasons to split `autoload_server` was so that it could be typically called with a single positional arg, also for debugging it's often nice to save computed args like this first, in case anyone ever needs to single step through the code. I'd suggest this:
```
url='http://%s:%d%s' % (notebook_url.split(':')[0], server.port, app_path)
script = server_document(url)
```